gatsby/src/components/button/button.tsx

```diff
@@ -41,6 +41,7 @@ type AnchorProps = React.AnchorHTMLAttributes<HTMLAnchorElement> &
   IButtonBase & { href: string };
 
 // Input/output options
+/* global JSX */
```

Comment:
Whats wrong with this? 🤔

Reply:
@rostaklein without that eslint throws an error `"'JSX' is not defined"  no-undef`